tests/tests_passing_pylint.txt

```diff
@@ -2,5 +2,7 @@ tests/data/test_dataset.py
 tests/data/test_dataset_access.py
 tests/ops/op_math/test_controlled_ops.py
 tests/ops/op_math/test_controlled.py
+tests/test_operation.py
```

Comment:
Was this one cleaned up already before?

Reply:
this came from merging with master, I didnt add it manually 🤔